docs/reference/rest-api/security/grant-api-keys.asciidoc

```diff
@@ -55,7 +58,7 @@ The following parameters can be specified in the body of a POST request:
 
 `access_token`::
 (Required*, string)
-The user's access token. If you specify the `access_token` grant type, this
+The user's {es} access token or JWT. If you specify the `access_token` grant type, this
```

Comment:
Can we add something to the effect - The final API key will be restricted to the permissions given to the user's {es} access token or JWT. (and similarly in the `username` field )

Also, not sure the wording...but should we specify this is JWT either access or ID tokens ? The re-use of access_token might be misleading so I think calling out JWT (access or id tokens) is warranted.

Reply:
I've changed it to:
```
The user's {es} access token, or JWT. Both access and id JWT token types
are supported, and they depend on the underlying JWT realm configuration.
The created API key will have a point in time snapshot of permissions of the
user authenticated with this token (or even more restricted permissions,
see the `role_descriptors` parameter).
If you specify the `access_token` grant type, this parameter is required.
It is not valid with other grant types.
```
WDYT?

Reply:
> The created API key will have a point in time snapshot of permissions of the
user authenticated with this token (or even more restricted permissions,
see the `role_descriptors` parameter).

 I would suggest to avoid "point in time" and "snapshot" since those have specific meanings in the context of ES. Perhaps 

> The created API key will be limited by permissions of the access_token (in addition to the `role_descriptors` parameter).